src/containers/SubmitSteps/Questionary/Step1c/Step1c.tsx

```diff
@@ -0,0 +1,196 @@
+
+  // Form
+  const {
+    control, handleSubmit, formState,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: state?.[storeKey],
+    context: {
+      country,
+    },
+    resolver: yupResolver(schemaWithoutPatient),
+  });
+  const { errors, isValid } = formState;
+
+  // Callbacks
+  const handleDoBack = React.useCallback(() => {
+    setActiveStep(false);
+    if (previousStep) {
+      history.push(previousStep);
+    } else {
+      history.goBack();
+    }
+  }, [history, previousStep]);
+
+  // Effects
+  useEffect(() => {
+    scrollToTop();
+    if (patientId) {
+      setTitle('');
+    } else {
+      setTitle(t('questionary:testTaken.title'));
+    }
+    if (patientId) {
+      setType('tertiary');
+    } else {
+      setType('primary');
+    }
+    setSubtitle('');
+    setDoGoBack(() => handleDoBack);
+  }, [handleDoBack, setDoGoBack, setTitle, setType, setSubtitle, patientId, t]);
+
+  // Handlers
+  const onSubmit = async (values: Step1cType) => {
+    if (values) {
+      const {
+        antigenTestDate,
+        antigenTestResult,
+      } = (values as any);
+      // if patient
+      if (!antigenTestDate || !antigenTestResult) {
+        return;
+      }
+
+      action(values);
+      if (nextStep) {
+        setActiveStep(false);
+        history.push(nextStep);
+      }
+    }
+  };
+
+  // Memos
+  const antigenOptions = React.useMemo(() => [
+    {
+      value: 'positive',
+      label: t('questionary:resultAntigenTest.options.positive'),
+    },
+    {
+      value: 'negative',
+      label: t('questionary:resultAntigenTest.options.negative'),
+    },
+  ], [t]);
+
+  return (
+    <MainContainer>
+      <ProgressIndicator
+        currentStep={metadata?.current}
+        totalSteps={metadata?.total}
+        progressBar
+      />
+      <QuestionText extraSpace first>
+        {t('questionary:whenAntigenTest')}
+        <QuestionAllApply>{t('questionary:whenAntigenTestNote')}</QuestionAllApply>
+      </QuestionText>
+
+      <Controller
+        control={control}
+        name="antigenTestDate"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <DatePicker
+            label="Date"
+            value={value ? new Date(value) : null}
+            locale={i18n.language}
+            onChange={onChange}
+          />
+        )}
+      />
+
+      <QuestionText extraSpace>
+        {t('questionary:resultAntigenTest.question')}
+      </QuestionText>
+      <Controller
+        control={control}
+        name="antigenTestResult"
+        defaultValue={undefined}
+        render={({ onChange, value }) => (
+          <OptionList
+            singleSelection
+            value={{ selected: value ? [value] : [] }}
+            onChange={v => onChange(v.selected[0])}
+            items={antigenOptions}
+          />
+        )}
+      />
+
+      {/* Bottom Buttons */}
+      <p><ErrorMessage errors={errors} name="name" /></p>
```

Comment:
This will add an empty space when there isn't errors for "name". Is the idea? Otherwise, use the render prop from ErrorMessage to return it.
Also, there is no "name" field here. Which error you will shown?